package.json

```diff
@@ -20,7 +20,8 @@
     "metadata": "pnpm -C tools/metadata cli",
     "slack:notify": "cd tools/slack && pnpm notify",
     "generate": "pnpm -C tools/generate cli generate",
-    "prerelease": "node scripts/prerelease.mjs --no-warnings"
+    "prerelease": "node scripts/prerelease.mjs --no-warnings",
+    "version:changeset": "pnpm changeset version --snapshot && node scripts/changeset-date.js"
```

Comment:
Does it work if this is just called `version`? Does pnpm pick our script over the default?

Reply:
@josephjclark  using `pnpm version` does not work